inputs/org.osbuild.containers

```diff
@@ -194,9 +231,25 @@ class ContainersInput(inputs.InputService):
         source = store.source("org.osbuild.containers")
         images = {}
 
+        storage_conf = containers.get_host_storage(None)
```

Comment:
(nitpick) I would just make the parameter optional in the definition of `containers.get_host_storage()`, i.e.:
```python
diff --git a/inputs/org.osbuild.containers b/inputs/org.osbuild.containers
index 41e56da3..ecb157d2 100755
--- a/inputs/org.osbuild.containers
+++ b/inputs/org.osbuild.containers
@@ -231,7 +231,7 @@ class ContainersInput(inputs.InputService):
         source = store.source("org.osbuild.containers")
         images = {}
 
-        storage_conf = containers.get_host_storage(None)
+        storage_conf = containers.get_host_storage()
 
         for ref, data in refs.items():
             if origin == "org.osbuild.source":
diff --git a/osbuild/util/containers.py b/osbuild/util/containers.py
index da7ce081..13830e4a 100644
--- a/osbuild/util/containers.py
+++ b/osbuild/util/containers.py
@@ -168,7 +168,7 @@ def container_source(image):
             yield image_name, image_source
 
 
-def get_host_storage(storage_conf):
+def get_host_storage(storage_conf=None):
     """
     Read the host storage configuration.
     """
```